src/components/grid/col.js

```diff
@@ -34,6 +34,15 @@ const Col = styled.div`
     `)}
   `}
 
+  ${p => p.order && css`
```

Comment:
```suggestion
  ${p => p.order !== undefined && css`
```

Reply:
Don't worry )) Work is work! ))
Wow... That's a good point ))

I fixed =)